api/src/main/java/org/openmrs/module/insuranceclaims/api/model/InsuranceClaimItem.java

```diff
@@ -56,7 +55,7 @@
 	@ManyToOne
 	@Cascade(CascadeType.SAVE_UPDATE)
 	@JoinColumn(name = "item", nullable = false)
```

Comment:
I know that this is created based on documentation by I am not sure if there should be nullable = false, need to be discussed with team

Reply:
according to our conversation with @kmadej, for now if we are not sure whether some field should be nullable or not, we should have these fields marked as nullable and if needed changes should be made later